samples/General/audit-allowed-locations/README.md

```diff
@@ -32,6 +32,6 @@ $assignment
 
 az policy definition create --name 'audit-location-deployments' --display-name 'Audit for allowed locations' --description 'This policy enables you to audit the locations where your resources have been deployed. Use this to understand what is within your environment and if it matches company guidelines.' --rules 'https://raw.githubusercontent.com/Azure/azure-policy/master/samples/Location/audit-allowed-locations/azurepolicy.rules.json' --params 'https://raw.githubusercontent.com/Azure/azure-policy/master/samples/Location/audit-allowed-locations/azurepolicy.parameters.json' --mode Indexed
 
-az policy assignment create --name <assignmentname> --scope <scope> --policy "audit-location-deployments" --params "{'listOfAllowedLocations':{'value': [ 'ukwest', 'eastus']}}"
+az policy assignment create --name <assignmentname> --scope <scope> --policy "audit-location-deployments" --params '{"listOfAllowedLocations":{'value': [ 'ukwest', 'eastus']}}'
```

Comment:
I've found params only work in CLI if you surround the entire thing in double quotes and use single quotes inside.  Does this work for you?